### PR TITLE
Project 4: add 'join table' for visits

### PR DESCRIPTION
**Pull request by Oklukeok**
Features:
-Created table vets.
-Created join table specializations.
-Created join table visits.
-Inserted data into vets table.
-Inserted data into specializations.
-Inserted data into visits.
-Wrote complex queries that answer analytical questions.

query1

query2

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -67,3 +67,25 @@ SELECT * FROM animals as Animals JOIN species as Species ON Species.id = Animals
...
 SELECT Animals.owner_id, Owners.full_name FROM animals as Animals INNER JOIN owners as Owners ON Animals.owner_id= Owners.id GROUP BY Animals.owner_id, Owners.full_name
 ORDER BY COUNT(*) DESC LIMIT 1;
+
+/*Project 4 */
+
+
+SELECT vets.name, animals.name, visit_date FROM visits INNER JOIN animals ON animals.id = visits.animals_id INNER JOIN vets ON vets.id = vets_id WHERE visit_date = (SELECT MAX(visit_date) FROM visits where vets_id= 1);
+
+
+SELECT COUNT(animals_id) FROM visits WHERE vets_id= 3;
```
Comment (AnselemOdims) [inline queries.sql]: - [ ] Good job writing the queries but the query is for `how many DIFFERENT animals did Stephanie Mendez see?`. The keyword here is DIFFERENT.
Kindly use a join to while writing this query.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Oklukeok, 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Oklukeok** [comment]
image

**Comment by Oklukeok** [comment]
Hi! the last print shows the correction based on last review.

**Review by KayLemba — APPROVED** [review]
[# STATUS: APPROVED :green_circle: :clap: 

Hello  @Oklukeok         , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- [x]  Your repo is professional 
- [x] Gitflow is followed
- [x] Descriptive P.R
- [x] Descriptive README file




 

- [x] No Linter Errors







Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]

**Pull request merged**